README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
```

Comment:
Please revert this change

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Make the code a little bit less clever in order to make it easier to read. Give it a stronger narrative structure so that you can more easily understand the code.

📄 [Sema Community]()

Reply:
Make the code a little bit less clever in order to make it easier to read. Give it a stronger narrative structure so that you can more easily understand the code.

📄 [Sema Community]()
Following consistent ordering for React.createClass: eslint: react/sort-comp

See Supporting Document for list.

📄 [AirBnB](https://github.com/airbnb/javascript/tree/master/react)

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
hkj

Reply:
TEST COMMENT

Reply:
TEST COMMENT

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
PERSONAL TEST

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient